src/components/SUE/SUEStatus.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { ViewStyle } from 'react-native';
+
+import { RegularText } from '../Text';
+import { Wrapper } from '../Wrapper';
+import { normalize } from '../../config';
+
+enum StatusTypes {
+  InBearbeitung = 'In Bearbeitung',
+  Offen = 'Offen',
+  Unbearbeitet = 'Unbearbeitet'
+}
+
+const defaultBackgroundColors = (status: string) => {
+  switch (status) {
+    case StatusTypes.InBearbeitung:
+      return '#FECDCA';
+    case StatusTypes.Offen:
+      return '#FEF0C7';
+    case StatusTypes.Unbearbeitet:
+      return '#D1FADF';
+    default:
+      return 'rgb(0, 0, 0)';
+  }
+};
+
+const defaultTextColors = (status: string) => {
+  switch (status) {
+    case StatusTypes.InBearbeitung:
+      return '#B42318';
+    case StatusTypes.Offen:
+      return '#B54708';
+    case StatusTypes.Unbearbeitet:
+      return '#027A48';
+    default:
+      return 'rgb(0, 0, 0)';
+  }
+};
+
+export const SUEStatus = ({
+  backgroundColors,
+  containerStyle,
+  status,
+  textColors
+}: {
+  backgroundColors: string;
+  containerStyle: ViewStyle;
+  status: string;
+  textColors: string;
+}) => {
+  const backgroundColor = backgroundColors?.[status] || defaultBackgroundColors(status);
+  const textColor = textColors?.[status] || defaultTextColors(status);
```

Comment:
this is not quite understandable. the type of `backgroundColors` and `textColors` are string but you access them as objects?
couldn't be values passed directly as string with fallbacks in method definition?

```
export const SUEStatus = ({
  backgroundColor = defaultBackgroundColors(status),
  containerStyle,
  status,
  textColor = defaultTextColors(status)
...
```

Reply:
Yeah, you're right. I should update the types.
if I build it this way, I don't get the status value I need. 
```
backgroundColor = defaultBackgroundColors(status),
...
```

Reply:
why is `backgroundColor` being passed as a prop and not part of `containerStyle`?
and then we can have another `textStyle` which can be passed. this way you can set the defaults which will be overwritten if given in `containerStyle` or `textStyle`.

Reply:
`containerStyle` is only one, but `backgroundColor` can be different for each `status`. If we add `backgroundColor` to `containerStyle`, we will need to write customised code for `backgroundColor` instead of standard style code. I don't think we want this to be the case.
It will then look as follows.

```
containerStyle: { 
"height:" 10, 
"padding": 20, 
backgroundColor: { "Offen": "#fff", "In Bearbeitung": "#dedede", "Unbearbeitet": "#000"}, 
textColor: { "Offen": "#fff", "In Bearbeitung": "#dedede", "Unbearbeitet": "#000"}}
```

Reply:
think about it again.
i mean something like:

```
export const SUEStatus = ({
  containerStyle,
  status,
...
const backgroundColor = backgroundColors?.[status] || defaultBackgroundColors(status);

style: { 
  "height:" 10, 
  "padding": 20, 
  backgroundColor
  ...containerStyle
```

this way there is a calculated background color based on the status and if we would pass a background color though `containerStyle` it would override it.

Reply:
Yes, that's right. I am talking about `globalSettings`. If we have one `backgroundColor`, all statuses will be the same color. That's not what we want, is it? We want a different color for each status. Accordingly, our style code in `globalSettings` will be far from standard.

I think it could be like this.
Code:
```
 style={{
        backgroundColor: containerStyle?.backgroundColor?.[status] || defaultBackgroundColors(status),
          alignItems: 'flex-end',
          alignSelf: 'flex-end',
          ...
          ...containerStyle
      }}
```

globalSettings: 
```
{
   ...
   containerStyle: {
      backgroundColor: { "Offen": "#fff", "In Bearbeitung": "#dedede", "Unbearbeitet": "#000"},
   }
}
```

Reply:
I have added a new section called `sueStatus` to `appDesignSystem` in `globalSettings`. Now we can update styles for `sueStatus` from there. 
https://github.com/smart-village-solutions/smart-village-app-app/pull/961/commits/8f01940116c2ceb51255795db12a0807249c399e